tests/rkt_list_test.go

```diff
@@ -95,3 +108,83 @@ func TestRktList(t *testing.T) {
 		runRktAndCheckOutput(t, runCmd, tt.expect, !tt.shouldSucceed)
 	}
 }
+
+func getCreationStartTime(t *testing.T, ctx *testutils.RktRunCtx, imageID string) (creation time.Time, start time.Time) {
+	// Run rkt list --full
+	rktCmd := fmt.Sprintf("%s list --full", ctx.Cmd())
+	child := spawnOrFail(t, rktCmd)
+	child.Wait()
+
+	// Get creation time
+	match := fmt.Sprintf(".*%s\t.*\t(.*)\t(.*)\t", imageID)
+	result, out, err := expectRegexWithOutput(child, match)
+	if err != nil {
+		t.Fatalf("%q regex not found, Error: %v\nOutput: %v", match, err, out)
+	}
+	tmStr := strings.TrimSpace(result[1])
+	creation, err = time.Parse(defaultTimeLayout, tmStr)
+	if err != nil {
+		t.Fatalf("Error parsing creation time: %q", err)
+	}
+
+	tmStr = strings.TrimSpace(result[2])
+	start, err = time.Parse(defaultTimeLayout, tmStr)
+	if err != nil {
+		t.Fatalf("Error parsing start time: %q", err)
+	}
+
+	return creation, start
+}
+
+func TestRktListCreatedStarted(t *testing.T) {
+	const imgName = "rkt-list-creation-start-time-test"
+
+	image := patchTestACI(fmt.Sprintf("%s.aci", imgName), fmt.Sprintf("--exec=/inspect --read-stdin"))
+	defer os.Remove(image)
+
+	imageHash := getHashOrPanic(image)
+	imgID := ImageID{image, imageHash}
+
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	// Prepare image
+	cmd := fmt.Sprintf("%s --insecure-skip-verify prepare %s", ctx.Cmd(), imgID.path)
+	podUuid := runRktAndGetUUID(t, cmd)
+
+	// t0: prepare
+	expectPrepare := time.Now()
+
+	// Get hash
+	imageID := fmt.Sprintf("sha512-%s", imgID.hash[:12])
+
+	tmpDir := createTempDirOrPanic(imgName)
+	defer os.RemoveAll(tmpDir)
+
+	time.Sleep(delta)
+
+	// Run image
+	cmd = fmt.Sprintf("%s run-prepared --interactive %s", ctx.Cmd(), podUuid)
+	rktChild := spawnOrFail(t, cmd)
+
+	// t1: run
+	expectRun := time.Now()
+
+	if err := expectWithOutput(rktChild, "Enter text:"); err != nil {
+		t.Fatalf("Expected a prompt but not found: %v", err)
+	}
+
+	say := "Hello"
+	if err := rktChild.SendLine(say); err != nil {
```

Comment:
We could skip `--read-stdin`, `--interactive` and this `expectWithOutput` + `SendLine` for this test since it is tested elsewhere.